src/frontend/components/TopNav.js

```diff
@@ -39,6 +40,17 @@ export default class TopNav extends React.Component {
     }
   }
 
+  logoutHandler = (event) => {
+    superagent
+      .post('/logout')
+      .end((err, res) => {
+        if (!err)
+          window.location = '/signup';
+        else
+          this.setState({errorMessage: 'Error logging out'});
```

Comment:
This setState won't do anything, I don't think.  It's fine for now to swallow this error though, but just so you are aware.  I'd probably explicitly make it a no-op (just 'return') so it's obvious to anyone else reading this code that this is not meant to do anything.